dist/api_data.py

```diff
@@ -813,6 +813,16 @@ def __ge__(self, other):
                 directory must already exist. If the value is not an absolute path, the path
                 is relative to the database home (see @ref absolute_path for more information)'''),
         ]),
+    Config('rollback_to_stable', '', r'''
+        rollback tables to an earlier point in time, discarding all updates to checkpoint durable
+        tables that have durable times more recent than the current global stable timestamp''',
+        type='category', subconfig=[
+            Config('threads', 4, r'''
+                maximum number of threads WiredTiger will start to help RTS. Each
+                RTS worker thread uses a session from the configured WT_RTS_MAX_WORKERS''',
+                min=0,
+                max=10),    # !!! Must match WT_RTS_MAX_WORKERS
```

Comment:
This is what I want. There is one other place to make the same `Must match...` change which is near (new) line 2025:
```
'WT_CONNECTION.rollback_to_stable' : Method([
    Config('dryrun', 'false', r'''
        perform the checks associated with RTS, but don't modify any data.''',
        type='boolean'),
    Config('threads', '4', r'''
        maximum number of threads WiredTiger will start to help RTS. Each
        RTS worker thread uses a session from the configured session_max''',
        min=0, max=10),
]),
```
That `10` needs the `Must match...` comment also please.